source/shared/cpp/ObjectModel/TextBlock.cpp

```diff
@@ -165,3 +168,126 @@ void TextBlock::SetHorizontalAlignment(const HorizontalAlignment value)
     m_hAlignment = value;
 }
 
+std::string TextBlock::ParseISO8601UsingRegex() const
+{
+	std::regex pattern("\\{\\{(DATE|TIME)\\((\\d{4})-{1}(\\d{2})-{1}(\\d{2})T(\\d{2}):{1}(\\d{2}):{1}(\\d{2})(Z|([+-])(\\d{2}):{1}(\\d{2}))(\\)|, S[Hh][Oo][Rr][Tt]\\)|, L[Oo][Nn][Gg]\\))\\}\\}");
```

Comment:
>	 [](start = 0, length = 1)

Looks like there are still tabs in this file

Reply:
There's still tabs in this iteration

---
In reply to: [149141278](https://github.com/Microsoft/AdaptiveCards/pull/871#discussion_r149141278) [](ancestors = 149141278)

Reply:
i guess there is something wrong with my vim setting.
let me look into that; thanks

---
In reply to: [149176815](https://github.com/Microsoft/AdaptiveCards/pull/871#discussion_r149176815) [](ancestors = 149176815,149141278)